gridstatus/utils.py

```diff
@@ -215,18 +215,26 @@ def get_response_blob(resp: requests.Response) -> io.BytesIO:
     return io.BytesIO(resp.content)
 
 
-def download_csvs_from_zip_url(url, process_csv=None, verbose=False):
+def download_csvs_from_zip_url(
+    url, process_csv=None, verbose=False, strip_whitespace_from_cols=False
+):
     z = get_zip_folder(url, verbose=verbose)
 
     all_dfs = []
+
     for f in z.filelist:
         if f.filename.endswith(".csv"):
             df = pd.read_csv(z.open(f.filename))
             if process_csv:
                 df = process_csv(df, f.filename)
+
+            if strip_whitespace_from_cols:
```

Comment:
maybe this can be removed if you move the stripping to `_process_hourly_load`?

Reply:
It needs to happen before the concat here based on the data. The data for some years has some files with whitespace in the column names and some without. If we leave the whitespace pandas thinks they are different columns and messes up the concat for the year. Unfortunately, I think we have to leave it here.